frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -176,28 +177,30 @@ async function checkTissues(
   expandedTissues = filteredTissues
 ) {
   if (filteredTissues.length !== 0) {
-    await expect(page.getByTestId("tissue-name")).toHaveCount(
+    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(
       filteredTissues.length
     );
   }
 
-  let i = 0;
+  let countExpanded = 0;
   for (const tissue of expandedTissues) {
     await expect(page.getByTestId(`cell-type-labels-${tissue}`)).toBeVisible();
     const height = await page
       .getByTestId(`cell-type-labels-${tissue}`)
       .getAttribute("height");
-    toInteger(height) > 20 && i++;
+    toInteger(height) > 20 && countExpanded++;
   }
-  await expect(i).toEqual(expandedTissues.length);
+  await expect(countExpanded).toEqual(expandedTissues.length);
```

Comment:
ooh that's why we're tracking the `i` 💡 Thanks for updating the var name!

Reply:
👍🏽